docs/OV_Runtime_UG/supported_plugins/GNA.md

```diff
@@ -10,371 +9,175 @@ to save power and free CPU resources.
+It is recommended to choose a compile target depending on what is a priority: cross-platform execution or performance, memory and power optimization.
 
-In the rest of this documentation, "GNA 2.0" refers to Intel® GNA hardware delivered on 10th and 11th generation Intel® Core™ processors, and the term "GNA 3.0" will be used to refer to GNA hardware delivered on 12th generation Intel® Core™ processors.
+You can use the following properties to check interoperability in your own application: `ov::intel_gna::execution_target` and `ov::intel_gna::compile_target`
 
-Initially, a limited subset of Intel® GNA 3.0 features are added to the previous feature set including the following:
+[Speech C++ Sample](@ref openvino_inference_engine_samples_speech_sample_README) can be used for experiments (see `-exec_target` and `-compile_target` command line options).
 
-* **2D VALID Convolution With Small 2D Kernels:**  Two-dimensional convolutions with the following kernel dimensions [H,W] are supported:  [1,1], [2,2], [3,3], [2,1], [3,1], [4,1], [5,1], [6,1], [7,1], [1,2], or [1,3].  Input tensor dimensions are limited to [1,8,16,16] <= [N,C,H,W] <= [1,120,384,240].  Up to 384 channels C may be used with a subset of kernel sizes (see table below).  Up to 256 kernels (output channels) are supported.  Pooling is limited to pool shapes of [1,1], [2,2], or [3,3].  Not all combinations of kernel shape and input tensor shape are supported (see the tables below for exact limitations).
+## Software emulation mode
 
-The tables below show that the exact limitation on the input tensor width W depends on the number of input channels C (indicated as Ci below) and the kernel shape.  There is much more freedom to choose the input tensor height and number of output channels.
+On platforms without GNA hardware support plugin chooses software emulation mode automatically by default. This means model will anyway run even if you don't have GNA HW within your platform.
+GNA plugin enables you to switch the execution between software emulation mode and hardware execution mode after the model is loaded.
+For details, see description of the `ov::intel_gna::execution_mode` property.
+
+## Recovery from Interruption by High-Priority Windows Audio Processes\*
+
+GNA is designed for real-time workloads such as noise reduction.
+For such workloads, processing should be time constrained, otherwise extra delays may cause undesired effects such as
+*audio glitches*. To make sure that processing can satisfy real-time requirements, the GNA driver provides a Quality of Service
+(QoS) mechanism, which interrupts requests that might cause high-priority Windows audio processes to miss
+the schedule, thereby causing long running GNA tasks to terminate early.
+
+Applications should be prepared for this situation. We recommend to use Automatic QoS Feature described below.
```

Comment:
To prepare the applications correctly, use Automatic QoS Feature described below.